examples/multimodel/conf/speechllm/modularized_speech_gpt_config.yaml

```diff
@@ -21,7 +21,6 @@ trainer:
   precision: 16
   logger: False # logger provided by exp_manager
   enable_checkpointing: False
-  replace_sampler_ddp: False
   max_epochs: 9999
```

Comment:
Does it work without adding `use_distributed_sampler: False`?

Reply:
No. If with `use_distributed_sampler: False`, will that affect our efficiency / precision in practice ?

Reply:
shouldn't affect, megatron uses something else I think